packages/init/bin/slicemachine-init.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+import("../dist/cli.cjs");
```

Comment:
are dynamic imports supported in all the node versions our users use?

Reply:
Yes, dynamic imports (`import()`) are supported as of Node.js 13.2.0. In Node.js, they are the only way to import ES modules from CJS code.

How far back are we supporting? I think Node.js 14 is a decent milestone to support.